### PR TITLE
recreated canvas medical webpage | James Roach

### PR DESCRIPTION
**Pull request by jamesroach7**
Screen Shot 2021-01-24 at 6 32 44 PM
Screen Shot 2021-01-24 at 6 32 55 PM

### CODE REVIEW

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  line-height: 1.5rem;
+}
+
+#text2,
+#text3 {
+  line-height: 2.5rem;
+}
+
+#text4 {
+  font-family: 'Avenir', sans-serif;
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:150]: If your body text is all `Avenir`, just set it on `body` once.

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  border: none;
+  border-radius: 0.2rem;
+  width: 10rem;
+  height: 2.7rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#gs {
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:176]: What does `gs` mean? Avoid abbreviations if you can.

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+#text1 {
+  font-size: 0.7rem;
+  line-height: 1.5rem;
+}
+
+#text2,
+#text3 {
+  line-height: 2.5rem;
+}
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:147]: What do `text1`, `text2`, and `text3` mean? Try to spend more time creating descriptive class names.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Looking really good, just needs some cleanup in the code.

File: css-canvas-medical/styles.css (hunk 3 above)
Comment (jamesroach7) [reply css-canvas-medical/styles.css:147]: Hey Tim! The main difference between them is their line height, so I'll try to take out the ID's and implement line height into the text classes with more descriptive names

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**